eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/ExceptionUtil.java

```diff
@@ -102,6 +102,9 @@ static Exception distinguishNotFound(final String message) {
             ((EventHubException) exception).setContext(errorContext);
         }
 
+        if (exception == null) {
```

Comment:
Looking at this method name: `ExceptionUtils.completeExceptionally`. Would we ever call this util method with an `exception == null`? 
It seems like it wouldn't be doing the right thing if we just returned rather than completing the future.

Reply:
updated with
   if (exception == null) {
            throw new NullPointerException();
   }
it just bring the error from ExceptionUtils.completeExceptionally  to this function